mycroft/configuration/mycroft.conf

```diff
@@ -81,6 +81,10 @@
   "skills": {
     // Directory to look for user skills
     "directory": "~/.mycroft/skills",
+    // blacklisted skills to not load
+    "blacklisted_skills": ["send_sms", "media"],
+    // priority skills to be loaded first
+    "priority_skills": [],
```

Comment:
yes we do... not in intent class but in other skills, i can give some personal use cases

my objectives skill, needs to listen to messages that can be emitted by any skill

some manifest skill that wants to listen to register messages of anything needs this

control center skill to enforce run levels at start up